src/AppInstallerRepositoryCore/SourceList.cpp

```diff
@@ -325,6 +331,7 @@ namespace AppInstaller::Repository
             details.Name = s_Source_DesktopFrameworks_Name;
             details.Type = Microsoft::PreIndexedPackageSourceFactory::Type();
             details.Arg = s_Source_DesktopFrameworks_Arg;
+            details.AlternateArg = s_Source_DesktopFrameworks_AlternateArg;

```

Comment:
Is it intentional to not check settings for this one?

Reply:
Yes, I was intentionally not affecting the platform one because it is used in an automated fashion.  If we had a GP, then I would have it affected then.